scripts/semconv/generate.sh

```diff
@@ -35,7 +36,19 @@ docker run --rm --platform linux/amd64 \
   registry generate \
   --registry=/source \
   --templates=/weaver/templates \
-  stable \
+  ts-stable \
+  /output/
+
+# Generate "semantic-conventions/src/experimental_*.ts".
+docker run --rm --platform linux/amd64 \
+  -v ${SCRIPT_DIR}/semantic-conventions/model:/source \
+  -v ${SCRIPT_DIR}/templates:/weaver/templates \
+  -v ${ROOT_DIR}/semantic-conventions/src/:/output \
+  otel/weaver:$GENERATOR_VERSION \
+  registry generate \
+  --registry=/source \
+  --templates=/weaver/templates \
+  ts-experimental \
```

Comment:
Review note: This PR splits the templates for stable and experimental to two separate dirs, so that the logic can differ between them.   Comment https://github.com/open-telemetry/opentelemetry-js/pull/5690#issuecomment-2885168882 includes a (slightly out of date) diff of the two dirs to show how similar they are.

I tried to share the unchanged files ("metrics.ts.j2", "docstring.ts.j2"), but weaver borks when you try to use files outside of the root dir.